CalcualtorLogic.js

```diff
@@ -30,13 +30,32 @@ class Calculator {
     };
     render() {
         const wrapperDiv = document.createElement('div');
-        const resultDiv = document.createElement('div');
         wrapperDiv.classList.add("wrapper");
+        const resultDiv = document.createElement('div');
         resultDiv.classList.add("result");
         resultDiv.id = "result";
         wrapperDiv.appendChild(resultDiv);
         this.hostElem.appendChild(wrapperDiv);
-        // console.log(this.hostElem); 
+        const div = document.createElement("div");
+        const button7 = document.createElement("button");
+        button7.dataset.number = "7";
+        button7.classList.add("button");
+        const button7Text = document.createDocumentFragment("7");
+        button7.appendChild(button7Text);
+        const button8 = document.createElement("button");
+        button8.dataset.number = "8";
+        button8.classList.add("button");
+        const button8Text = document.createDocumentFragment("8");
+        button8.appendChild(button8Text);
+        const button9 = document.createElement("button");
+        button9.dataset.number = "9";
+        button9.classList.add("button");
+        const button9Text = document.createDocumentFragment("9");
+        button9.appendChild(button9Text);
+        div.appendChild(button7);
+        div.appendChild(button8);
+        div.appendChild(button9);
```

Comment:
After adding `button9` to the `div`; please append the new div to the `wrapeprDiv`